### PR TITLE
Heroku init

### PR DESCRIPTION
**Pull request by tuna42na**

### CODE REVIEW

**Comment by tuna42na** [comment]
@trvslhlt I just wanna check with you on that this is the correct way of doing this.

I'm pushing the Heroku changes, which is essentially just making an express server, but I want to make sure that it maintains the unit testing changes that you've made. 

Will it automatically know not to override those changes? It seems like it's only telling me that there are only 3 files changed, but I don't wanna merge if it'll over write what you've done.

**Comment by trvslhlt** [comment]
> @trvslhlt I just wanna check with you on that this is the correct way of doing this.
> 
> I'm pushing the Heroku changes, which is essentially just making an express server, but I want to make sure that it maintains the unit testing changes that you've made.
> 
> Will it automatically know not to override those changes? It seems like it's only telling me that there are only 3 files changed, but I don't wanna merge if it'll over write what you've done.

This PR shouldn't overwrite the unit-test/Chart changes recently merged to master. None of the changes in this PR were made in the same files modified in the last PR. But, it is good to get in the habit of making sure feature branches are up to date with master before merging them in. If you check out the [repo's branches](https://github.com/tuna42na/baby-namr/branches) you will see that `heroku-init` is one commit behind master. You can address this by:
1. On your computer, run `git checkout master`
2. `git pull`. make sure your local master branch has the most recent commits on the remote branch.
3. `git checkout heroku-init`
4. `git merge master`. make sure that the most recent master commits are on your feature branch
5. `git push`. Check the branches again on github to make sure that your feature branch is caught up with master.

File: server.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var path = require("path");
+var express = require("express");
+
+var app = express();
+
+app.use(express.static(path.join(__dirname, "dist")));
```
Comment (trvslhlt) [inline server.js:6]: When is the web app built and put in the dist directory?

Comment (tuna42na) [reply server.js:6]: The rules for this area in the base webpack configuration that is run with the build script. I'm not sure exactly what is being used to compress and compile it, but I know that heroku runs the build and starts scripts upon initialization.

**Pull request merged**